### PR TITLE
Changes for ibm xlf compiler compatibility

### PR DESCRIPTION
**Pull request by cnegre**
Changes for xlf IBM compiler compatibility.

---
This change is [Reviewable](https://reviewable.io/reviews/lanl/qmd-progress/128)

### CODE REVIEW

File: src/prg_normalize_mod.F90 [hunk 1]
```diff
@@ -128,12 +129,14 @@ end subroutine prg_gershgorinReduction
...
   subroutine prg_normalize_cheb(h_bml,mu,emin,emax,alpha,scaledmu)
 
+    implicit none
     type(bml_matrix_t),intent(inout) :: h_bml
 
     real(dp) :: beta, maxMinusMin
     real(dp), intent(in) :: mu, emin, emax
     real(dp), intent(inout) :: scaledmu, alpha
 
+    stop
```
Comment (nicolasbock) [inline src/prg_normalize_mod.F90]: Is that a left over debug statement, or do you want to leave it here?

File: tests/src/main.F90 [hunk 2]
```diff
@@ -895,7 +896,7 @@ program main
   case("prg_system_parse_write_xyz")
      call prg_parse_system(mol,"coords_100","xyz")
      call prg_write_system(mol, "mysystem","xyz")
-     call system("diff -qs  mysystem.xyz coords_100.xyz > tmp.tmp")
+     call system("diff -qs --ignore-space-change mysystem.xyz coords_100.xyz > tmp.tmp")
```
Comment (nicolasbock) [inline tests/src/main.F90]: Alternatively, you could use `--ignore-all-space` which also includes changes in trailing whitespace.

File: src/prg_normalize_mod.F90 (hunk 1 above)
Comment (cnegre) [reply src/prg_normalize_mod.F90]: Good catch Nick, thanks!

File: tests/src/main.F90 (hunk 2 above)
Comment (cnegre) [reply tests/src/main.F90]: I'll make that change, thanks!

File: tests/src/main.F90 [hunk 3]
```diff
@@ -934,7 +935,8 @@ program main
      !> Loading the tb parameters (electrons.dat)
      call load_latteTBparams(tbparams,mol%splist,"./")
      call write_latteTBparams(tbparams,"myelectrons.dat")
-     call system("diff -qs  myelectrons.dat electrons.dat > tmp.tmp")
+     call system("diff -qs --ignore-all-space myelectrons.dat electrons.dat > tmp.tmp")
+     call system("diff  --ignore-all-space myelectrons.dat electrons.dat")
```
Comment (nicolasbock) [inline tests/src/main.F90]: If you would like to print the `diff` _and_ write it to file you can do this in one line with
```
call system("diff --report-identical-files --ignore-all-space myelectrons.dat electrons.dat | tee tmp.tmp")
```

Comment (nicolasbock) [reply tests/src/main.F90]: I have expanded the `-s` option to `--report-identical-files` which I personally find more readable. But that's not necessary for the above to work.

**Pull request merged**